doctrine.rst

```diff
@@ -412,6 +412,72 @@ Take a look at the previous example in more detail:
 Whether you're creating or updating objects, the workflow is always the same: Doctrine
 is smart enough to know if it should INSERT or UPDATE your entity.
 
+Validating Objects
+------------------
+
+:doc:`The Symfony validator </validation>` reuse Doctrine metadata
+to perform some basic validation tasks::
```

Comment:
Let's remove the `basic` word in this phrase. Thanks.

Reply:
IMO it's not problematic here because that's exactly what it is: basic validation rules derived from metadata.

If you still insist, maybe reword as
> reuses Doctrine metadata to establish a baseline of validation rules.